app/gateway/traditional-mode.md

```diff
@@ -0,0 +1,158 @@
+the node purged it from its local cache.
+
+## Multiple node clusters
+
+In a cluster of multiple {{site.base_gateway}} nodes, other nodes connected to the same database
+wouldn't be instantly notified that the Service was deleted by node `A`.  While
+the Service is **not** in the database anymore (it was deleted by node `A`), it is
+**still** in node `B`'s memory.
+
+All nodes perform a periodic background job to synchronize with changes that
+may have been triggered by other nodes. The frequency of this job can be
+configured using the [`db_update_frequency`](/gateway/configuration/#db_update_frequency) parameter in `kong.conf`.
+
+Every `db_update_frequency` seconds, all running {{site.base_gateway}} nodes will poll the
+database for any update, and will purge the relevant entities from their cache
+if necessary.
+
+If we delete a Service from node `A`, this change will not be effective in node
+`B` until node `B`'s next database poll, which will occur up to
+`db_update_frequency` seconds later (though it could happen sooner).
+
+This makes {{site.base_gateway}} clusters **eventually consistent**.
+
+## Using read-only replicas when deploying {{site.base_gateway}} clusters with PostgresSQL
+
+When using Postgres as the backend storage, you can optionally enable
+{{site.base_gateway}} to serve read queries from a separate database instance.
+
+Enabling the read-only connection support in {{site.base_gateway}}
+greatly reduces the load on the main database instance since read-only
+queries are no longer sent to it.
+
+To learn more about how to configure this feature, refer to the
+[Datastore section](/gateway/configuration/#datastore-section)
+of the {{site.base_gateway}} configuration reference.
+
+## What information is cached?
+
+For performance reasons, {{site.base_gateway}} avoids database connections when proxying
+requests, and caches the contents of your database in memory. The following entities are cached:
+
+* [Gateway Services](/gateway/entities/service/)
+* [Routes](/gateway/entities/route/)
+* [Consumers](/gateway/entities/consumer/)
+* [Plugins](/gateway/entities/plugin/)
+* Credentials, and so on 
+
+Since these values are in memory, any change made via the [Admin API](/api/gateway/admin-ee/#/operations/) of one of the nodes must be propagated to the other nodes.
+
+Additionally, {{site.base_gateway}} also caches **database misses**. This means that if you
+configure a Service with no plugin, {{site.base_gateway}} will cache this information. 
+
+All CRUD operations trigger cache invalidations. Creation
+(`POST`, `PUT`) will invalidate cached database misses, and update/deletion
+(`PATCH`, `DELETE`) will invalidate cached database hits.
+
+## Configure database caching
+
+You can configure three properties in the {{site.base_gateway}} configuration file, the most
+important one being `db_update_frequency`, which determine where your {{site.base_gateway}}
+nodes stand on the performance versus consistency trade-off.
```

Comment:
```suggestion
Using parameters in `kong.conf`, you can configure where {{site.base_gateway}} nodes stand on the performance versus consistency trade-off.
```
I don't know that you need the "most important" statement when there's really only three of them, and readers can see all the details.